docs/pbc/all/product-information-management/202212.0/install-and-upgrade/install-features/install-the-measurement-units-feature.md

```diff
@@ -1004,7 +1005,7 @@ class ProductDetailPageDependencyProvider extends SprykerShopProductDetailPageDe
     protected function getProductDetailPageWidgetPlugins(): array
     {
         return [
-            ProductMeasurementUnitWidgetPlugin::class,
```

Comment:
Removed since `spryker-shop/product-measurement-unit-widget:1.0.0`:
* https://github.com/spryker-shop/product-measurement-unit-widget/releases/tag/1.0.0